shared/src/common/types/responses/login-response.types.ts

```diff
@@ -0,0 +1,8 @@
+import { IUser } from "~/common/interfaces";
+
+type LoginResponse = {
+  token: string,
+  user: IUser | null
```

Comment:
```suggestion
  user: IUser
```
When we logged we alway have a user, isn't it?